src/test/java/org/opensearch/security/auditlog/AbstractAuditlogiUnitTest.java

```diff
@@ -56,7 +56,7 @@ protected final void setup(Settings settings) throws Exception {
         });
 
         final Settings nodeSettings = defaultNodeSettings(defaultNodeSettings.build());
-        setup(Settings.EMPTY, new DynamicSecurityConfig(), nodeSettings, init);
+        setup(Settings.EMPTY, securityConfig, nodeSettings, init);
```

Comment:
@dancristiancecoi is the CI failure related to this changes in this file?

ref: https://github.com/opensearch-project/security/actions/runs/9793063749/job/27040225350?pr=4524

```
  2> java.lang.NullPointerException: Cannot invoke "org.opensearch.security.test.helper.rest.RestHelper.executeDeleteRequest(String, org.apache.hc.core5.http.Header[])" because "this.rh" is null
        at __randomizedtesting.SeedInfo.seed([C16759D4AFC08567:D86111E1CD25905D]:0)
        at org.opensearch.security.auditlog.integration.BasicAuditlogTest.testIndexCloseDelete(BasicAuditlogTest.java:775)
```

Reply:
Potentially, i will have a look! Would it be possible to re-run that test run to make sure its not an intermittent failure? (

Reply:
Sure, I just triggered a re-run. I have never seen that failure before so I think it is directly related to this PR.

Reply:
Thank you! Does look like its an issue with the PR. I will investigate